controllers/user.js

```diff
@@ -260,4 +260,26 @@ export default class UserController {
       });
     }
   }
+
+  /**
+   * @description Get user's article count
+   * @param {object} req http request object
+   * @param {object} res http response object
+   * @returns {object} response
+   */
+  static async getUserArticlesCount(req, res) {
+    const { id: userId } = req.user;
+    try {
+      const { count } = await Article.findAndCountAll({ where: { userId } });
+      return res.status(200).json({
+        success: true,
+        numberOfArticlesCreated: count
```

Comment:
@kleva-j can you come up with a better name here?

Reply:
Ok, thanks.